pkg/api/v1beta1/conversion.go

```diff
@@ -579,6 +579,9 @@ func init() {
 			if err := s.Convert(&in.Capabilities, &out.Capabilities, 0); err != nil {
```

Comment:
This conversion doesn't need to be called this way anymore, since this is a field on `SecurityContext` now.